src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -1156,7 +1156,14 @@ public LocalDate fromString(String string) {
         c -> new SimpleStringProperty(
             Provider.getByInternalLabel(c.getValue().getProvider()).toString())
     );
-
+    tableColumnCvrContestId.setCellValueFactory(new PropertyValueFactory<>("contestId"));
+    tableColumnCvrOvervoteLabel.setCellValueFactory(new PropertyValueFactory<>("overvoteLabel"));
+    tableColumnCvrSkippedRankLabel
+        .setCellValueFactory(new PropertyValueFactory<>("skippedRankLabel"));
+    tableColumnCvrUndeclaredWriteInLabel
+        .setCellValueFactory(new PropertyValueFactory<>("undeclaredWriteInLabel"));
+    tableColumnCvrTreatBlankAsUndeclaredWriteIn
+        .setCellValueFactory(new PropertyValueFactory<>("treatBlankAsUndeclaredWriteIn"));
```

Comment:
could you clarify why these changes are in this PR and what they do?

Reply:
I think github is confused!! I don't think these are actually in this PR -- let me merge develop and see what happens. Not sure why this is happening. These are part of editable table changes.

Reply:
Merging develop didn't help, but locally, `git blame` shows that you added these lines in 2020 and the only changes are to `tableColumnCvrSkippedRankLabel`. I'm very confused here too -- I merged develop and it didn't go away. Is this a github bug or is there something I'm missing? If it's a github bug, it'll make reviewing very hard...

Reply:
Bizarre. No idea!